docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx

```diff
@@ -573,6 +580,31 @@ Request Reviewed` in the Teleport Web UI.
 
 </Admonition>
 
+### Set up notification alerts
```

Comment:
Would you mind adding some context for why this new section is here? 

Does the `editor-requester` role created in Step 2 already achieve this?

Reply:
The request https://github.com/gravitational/teleport/issues/29651 called this out:
> [plugin readme](https://github.com/gravitational/teleport-plugins/blob/v13.2.3/access/pagerduty/README.md)
> has additional information not in the guide, like [how to set up a pagerduty notification alert](https://github.com/gravitational/teleport-plugins/blob/v13.2.3/access/pagerduty/README.md#setting-up-pagerduty-notification-alerts)

I can delete this section if Step 2 covers it sufficiently.